tests/Http/Three/SignupTest.php

```diff
@@ -5,10 +5,83 @@
 use Rogue\Models\Post;
 use Rogue\Models\Signup;
 use Tests\BrowserKitTestCase;
+use DoSomething\Gateway\Blink;
 
 class SignupTest extends BrowserKitTestCase
 {
     /**
+     * Test that a POST request to /signups creates a new signup.
+     *
+     * POST /api/v3/signups
+     * @return void
+     */
+    public function testCreatingASignup()
+    {
+        $northstarId = '54fa272b469c64d7068b456a';
```

Comment:
Let's use `$this->faker->northstar_id` and `$this->faker->campaign_id` for these two!